git-rebase-via-merge.sh

```diff
@@ -113,7 +113,7 @@ init() {
 
   while true; do
     echo "Continue (c) / Abort (a)"
-    read -r input
+    read input
```

Comment:
Looks like we don't need `-r` flag here. I would like to keep the script as simple as possible.

Reply:
That was just a shellcheck linter warning: https://www.shellcheck.net/wiki/SC2162

I agree though because we don't care about backslashes. Will remove!